src/main/java/dk/kb/util/BuildInfoManager.java

```diff
@@ -85,10 +129,62 @@ public static String getBuildTime() {
         return buildTime;
     }
 
+    /**
+     * @return name of the built branch.
+     */
+    public static String getGitBranch(){
+        if (gitBranch == null){
+            loadBuildInfo();
+        }
+        return gitBranch;
+    }
+
+
+    public static String getGitCommitChecksum(){
+        if (gitCommitChecksum == null){
+            loadBuildInfo();
+        }
+        return gitCommitChecksum;
+    }
+
+    public static String getGitCommitTime(){
+        if (gitCommitTime == null ){
+            loadBuildInfo();
+        }
+        return gitCommitTime;
+    }
+
+    public static String getGitCurrentTag(){
+        if (gitCurrentTag == null){
+            loadBuildInfo();
+        }
+        return gitCurrentTag;
+    }
+
+    public static String getGitClosestTag(){
+        if (gitClosestTag == null){
+            loadBuildInfo();
+        }
```

Comment:
Afterthought: The `loadBuildInfo` is silent and a no-op when the properties has already been loaded so there is no harm done in just calling it every time instead of checking if it needs to be called. Removing the `null`-check would make the code cleaner.